### PR TITLE
feat: add logos

### PR DESCRIPTION
**Pull request by jannyHou**
Signed-off-by: jannyHou <juehou@ca.ibm.com>

Add two more companies: https://www.codewave.com/ and https://dayoneteams.com/

Screen Shot 2020-11-09 at 1 09 20 PM

### CODE REVIEW

**Review by dhmlau — APPROVED** [review]
Thanks @jannyHou for updating this. 
Just in case, there's a testimonial from CodeWave as well. Feel free to update it in a separate PR. Thanks.

File: what-our-users-say.html [hunk 1]
```diff
@@ -242,6 +242,14 @@ <h5 style="font-weight: normal">Let's continue to build our community together.<
                     </a>
                 </div>
                 <div class="col-sm-12 col-md-6">
+                    <a class="card-2">
+                    <div class="row" style="padding:10px">
+                        "At Codewave, we help SME’s and Enterprises across the globe, modernise their existing applications or build robust new innovative digital initiatives from the ground up and Loopback has "got our back" on more than one occasions, bringing rapid innovation without compromising on security or scalability."
```
Comment (dhmlau) [inline what-our-users-say.html]: Minor edits:
```suggestion
                        "At Codewave, we help SME’s and Enterprises across the globe, modernise their existing applications and build robust new innovative digital initiatives from the ground up and LoopBack has "got our back" on more than one occasions, bringing rapid innovation without compromising on security or scalability."
```

**Comment by jannyHou** [comment]
The new entry:

Screen Shot 2020-11-10 at 11 24 16 AM

And the updated entry:
Screen Shot 2020-11-10 at 11 24 54 AM

File: what-our-users-say.html (hunk 1 above)
Comment (jannyHou) [reply what-our-users-say.html]: @dhmlau thanks fixed. And the new entry is also added.

File: what-our-users-say.html [hunk 2]
```diff
@@ -247,7 +247,19 @@ <h5 style="font-weight: normal">Let's continue to build our community together.<
...
+                </div>
+            </div>
+            <div class="row text-center" style="justify-content: center">
+                <div class="col-sm-12 col-md-6">
+                    <a class="card-2">
+                    <div class="row" style="padding:10px">
+                        "At DayOne, we put customers first and have a strong commitment to deliver the best quality technology solutions with fast-pace velocity. Therefore, a reliable, secured, well-maintained, powerful and performant back-end framework is must! Loopback just simply exceeds our expectation! Having successfully built and delivered dozens of digital products and platforms for e-learning, crypto, e-commerce, food delivery, travel, real estate platforms and many digital transformation projects for both startups and enterprises using LoopBack, we know this is still gonna be our go-to framework to build further amazing products in the future! Using LoopBack for year from the early day of version 2 until current version 4, I also highly appreciate the dedication of LoopBack team for their passion, long-term commitment, and caring of the community for a very long time. Thank you so much!"
```
Comment (dhmlau) [inline what-our-users-say.html]:
```suggestion
                        "At DayOne, we put customers first and have a strong commitment to deliver the best quality technology solutions with fast-pace velocity. Therefore, a reliable, secured, well-maintained, powerful and performant back-end framework is must! LoopBack just simply exceeds our expectation! Having successfully built and delivered dozens of digital products and platforms for e-learning, crypto, e-commerce, food delivery, travel, real estate platforms and many digital transformation projects for both startups and enterprises using LoopBack, we know this is still gonna be our go-to framework to build further amazing products in the future! Using LoopBack for year from the early day of version 2 until current version 4, I also highly appreciate the dedication of LoopBack team for their passion, long-term commitment, and caring of the community for a very long time. Thank you so much!"
```

Comment (jannyHou) [reply what-our-users-say.html]: @dhmlau I updated all `Loopback` in the new commit :)

**Review by dhmlau — APPROVED** [review]

**Pull request merged**